### PR TITLE
Rewrote Fedora single- and multi-node Kube setup docs

### PR DESCRIPTION
**Pull request by chrisnegus**
The two Fedora kube [single-node](https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/) and [multi-node](https://kubernetes.io/docs/getting-started-guides/fedora/flannel_multi_node_cluster/) getting started guides were outdated. These two issues suggested the two docs be updated to use kubeadm:

[#7302](https://github.com/kubernetes/website/issues/7302)
[#7278](https://github.com/kubernetes/website/issues/7301)

I totally rewrote both of those documents to use kubeadm.

(Note: This is my first PR so please let me know if I did anything wrong.)

### CODE REVIEW

**Comment by hogepodge** [comment]
Hi @chrisnegus, one of the things the wg-cloud-provider will be working on is a general location for smaller providers and documentation that would be managed by the wg, but contributed to by third party maintainers. Feel free to drop in on the #wg-cloud-provider slack channel or our regular meetings on Tuesday at 10 PT https://github.com/kubernetes/community/tree/master/wg-cloud-provider#meetings

File: docs/getting-started-guides/fedora/fedora_manual_config.md [hunk 1]
```diff
@@ -1,185 +1,228 @@
...
+    <pre><tt>
+    # <b>swapoff -a</b>
+    # <b>setenforce 0</b>
+    </tt></pre>
 
-```shell
-echo "192.168.121.9    fed-master
-192.168.121.65    fed-node" >> /etc/hosts
-```
+5. Disable firewalld. On Fedora Server or Workstation only (firewalld is no on Fedora Atomic by default), type the following.
```
Comment (hogepodge) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: nit, replace "no on" with "not enabled on"

File: docs/getting-started-guides/fedora/fedora_manual_config.md [hunk 2]
```diff
@@ -1,185 +1,228 @@
...
+    <pre><tt>
+    # <b>systemctl stop firewalld</b>
+    # <b>systemctl disable firewalld</b>
+    </tt></pre>
 
-```shell
-# Comma separated list of nodes in the etcd cluster
-KUBE_MASTER="--master=http://fed-master:8080"
-```
+**WARNING**: Although disabling security features, such as SELinux and Firewalld, are currently needed to get this procedure to work properly, disabling them is not good practice when using Kubernetes in production. For that reason, this procedure is best used for trying and learning about Kubernetes in non-production environments.
```
Comment (hogepodge) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: This deserves additional information. A pointer to a more thorough explanation of how to run these services in production with SELinux and Firewalld is fairly important if you really want behaviors around those security services to change. I want to run SELinux, I just never really see great documentation that helps me be successful in running SELinux

File: docs/getting-started-guides/fedora/fedora_manual_config.md [hunk 3]
```diff
@@ -1,185 +1,228 @@
...
+    nginx-7587c6fdb6-6nzq8 0/1 Running  0        10s 10.244.0.4 fedoraatomic27
+    nginx-7587c6fdb6-7xssr 0/1 Running  0        10s 10.244.0.5 fedoraatomic27
+    </tt></pre>
 
-```shell
-kubectl get nodes
-NAME            STATUS      AGE      VERSION
-fed-node        Ready       4h
-```
+3. Expose nginx ports to they are accessible from the host:
```
Comment (hogepodge) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: to -> so

File: docs/getting-started-guides/fedora/fedora_manual_config.md (hunk 1 above)
Comment (chrisnegus) [reply docs/getting-started-guides/fedora/fedora_manual_config.md]: Good catch. It should be "is not enabled on Fedora".

File: docs/getting-started-guides/fedora/fedora_manual_config.md (hunk 2 above)
Comment (chrisnegus) [reply docs/getting-started-guides/fedora/fedora_manual_config.md]: I wasn't thinking of using kubeadm to set up a production-ready Kubernetes system. I thought it was a way to try out Kubernetes in a demo type of environment. With firewalld, I know at one point it caused problems with the way that Kubernetes used port forwarding. I suppose I could just set up some iptables firewall rules instead. With SELinux, I could create an SELinux policy, but that might require some explanation. Also, the kubeadm approach doesn't address issues like setting up storage or DNS. Do you think I need to add these features before publishing? Or could they be skipped or added as enhancements later?

File: docs/getting-started-guides/fedora/fedora_manual_config.md (hunk 3 above)
Comment (chrisnegus) [reply docs/getting-started-guides/fedora/fedora_manual_config.md]: Good catch again. The word should be "so".

Comment (chrisnegus) [reply docs/getting-started-guides/fedora/fedora_manual_config.md]: Thanks for the feedback @hogepodge . I'll try to make your Cloud Provider Refactoring WG on Wednesday. Is that group limited to cloud providers or is it for operating system providers as well?

**Comment by zacharysarah** [comment]
@chrisnegus 👋 Any updates here?

**Comment by chrisnegus** [comment]
@zacharysarah I'd like to accept the small changes suggested above. I attended the wg-cloud-provider meeting and their work didn't seem appropriate to these two documents. Can we then publish this pull request?

**Comment by zacharysarah** [comment]
@chrisnegus 👋 We've migrated the site's framework to Hugo. Please re-open this PR with changes to files in their post-migration location (`/content/en/docs/...`). 

(☝️ Boilerplate aside, yes, I think we can go ahead and merge this PR once you re-open it. Please be sure to reference this PR in the description of the new one.)

**Pull request closed without merging**